### PR TITLE
add simple container definition

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM postgres:12
+
+COPY Makefile *.control *.sql /src
+
+RUN apt-get update && apt-get -y -qq install make && make -C /src install
```
Comment (Odilhao) [inline Containerfile]: `apt-get clean` :eyes:

Comment (evgeni) [reply Containerfile]: go back to your snakes!

**Comment by evgeni** [comment]
example action run: https://github.com/evgeni/postgresql-evr/actions/runs/6957928680/job/18931797270

**Review by Odilhao — APPROVED** [review]

**Review by ekohl — APPROVED** [review]
Any thoughts on publishing this too, at least to the GH container registry? That way we can use it in other actions as well.

**Comment by evgeni** [comment]
> Any thoughts on publishing this too, at least to the GH container registry? That way we can use it in other actions as well.

This is prepared as a follow up, yes.

**Review by ehelms — APPROVED** [review]

**Pull request merged**